.github/CODE_OF_CONDUCT.md

```diff
@@ -1,49 +1,49 @@
-## Code of Conduct
+# Contributor Covenant Code of Conduct
 
-### Our Pledge
+## Our Pledge
 
 In the interest of fostering an open and welcoming environment, we as
 contributors and maintainers pledge to making participation in our project and
 our community a harassment-free experience for everyone, regardless of age, body
-size, disability, ethnicity, gender identity and expression, level of experience,
-nationality, personal appearance, race, religion, or sexual identity and
-orientation.
+size, disability, ethnicity, sex characteristics, gender identity and expression,
```

Comment:
I *think* `sex characteristics`, `gender identity`, and `sexual identity` are all very similar. I could be wrong though?

Reply:
I didn't write this. I just replaced the old one with what GitHub provides/suggests

Reply:
Ahh, okay. They know more than me!